test/e2e/network/netpol/model.go

```diff
@@ -21,11 +21,28 @@ import (
 	"strings"
 
 	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/kubernetes/test/e2e/framework"
 	imageutils "k8s.io/kubernetes/test/utils/image"
 )
 
+const (
+	CPURequest    = "1m"
+	MemoryRequest = "50Mi"
+	CPULimit      = "75m"
```

Comment:
Why not use `resource.MustParse` here so you don't have to parse resources for every single container?

Reply:
Hmmm you can do that in a constant? Not sure what the concern is here anyways ....

Reply:
..... yeah , confirmed, i dont think you *can* call a function from a golang constant.... ?

Reply:
You can use an `init` function if necessary.

Reply:
Is there an example elsewhere in k8s tests that you wanted rajas to follow ?

Reply:
https://golang.org/doc/effective_go#init